tools/pubsys-config/src/lib.rs

```diff
@@ -26,6 +26,9 @@ pub struct InfraConfig {
 
     // Config for VMware specific subcommands
     pub vmware: Option<VmwareConfig>,
+
+    // Kit Vendors for publishing kits
+    pub kit_vendors: Option<HashMap<String, String>>,
```

Comment:
Should we call this `vendors` in case we need to push things other than just kits someday?